test/Makefile

```diff
@@ -0,0 +1,28 @@
+MAKEFLAGS += --output-sync=target
+
+tests = \
+	metadata \
+	pause \
+	play \
+	play-pause \
+	stop
+
+.PHONY: \
+	test \
+	$(tests) \
+	clean
+
```

Comment:
The tests fail if running make with parallel jobs. Took me a while to realise the tests were failing on my machine because I have a terminal alias for make=make -j24

The easy solution would be to add a [.NOTPARALLEL](https://www.gnu.org/software/make/manual/make.html#index-_002eNOTPARALLEL) directive for these test jobs or fix running them in parallel.

Reply:
The tests work in parallel for me, how are you running them? I use this:

```
make -j10 test
```

Each test should be using independent files/dirs/sockets so I'm surprised they break in parallel for you.

Reply:
running `make -j10 test` on my machine gives errors like these at the end of some tests:

```
/usr/bin/xvfb-run: line 186: kill: (5825) - No such process
test play-pause: stderr not empty!
test play-pause: start of stderr ---------------------
/usr/bin/xvfb-run: line 186: kill: (5825) - No such process
test play-pause: end of stderr -----------------------
test play-pause: stderr not empty!
test play-pause: failed with exit code 1
make[1]: *** [Makefile:18: play-pause] Error 1
make[1]: Leaving directory '/home/hoyon/code/mpv-mpris/test'
make: *** [Makefile:56: test] Error 2
```

Possibly another bug in the arch linux xvfb-run.

Reply:
Thanks, I'll try to download a copy of their xvfb-run and then run it on Debian.

Reply:
BTW, can you try a parallel test with the new version that fixed the bug you linked above?

Reply:
OK, I reproduced the failure with their version.

Reply:
OK, switching it to use `--auto-display` instead of `--auto-servernum` makes it work. Unfortunately the Debian `xvfb-run` does not have `--auto-display` though, but the `--auto-servernum` works fine with it. So I think that the right approach is to grep the `--help` output and use `--auto-display` when it is present.

Reply:
Implemented that, fixes the parallel failure and works on Debian too.